.packit/rpmlint.spec

```diff
@@ -0,0 +1,88 @@
+%{!?python3: %global python3 %{__python3}}
+
+%bcond_with tests
+
+Name:           rpmlint
+Version:        2.0.0
+Release:        0%{?dist}
+Summary:        Tool for checking common errors in RPM packages
+
+License:        GPLv2+
+URL:            https://github.com/rpm-software-management/rpmlint
+Source0:        %{url}/archive/%{name}-%{version}.tar.gz
+
+BuildArch:      noarch
+
+BuildRequires:  python3-devel
+BuildRequires:  python3dist(setuptools)
```

Comment:
this cant work on openSUSE

Reply:
This will be fixed in openSUSE soon enough, so I'm not concerned. It's not getting merged right now anyway.

Reply:
Joke?  we don't want crystal ball dependencies generator with Bulgarian constant...

Reply:
That's not a joke. The generator exists in openSUSE Tumbleweed now. Most things already have the auto-generated Provides, just only not things in Tumbleweed ring 0.